client/app/components/directions.jsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import Legend from './Legend.jsx';
+import Legend from './legend.jsx';
```

Comment:
Whoa. Just realized none of our component files are capitalized. I believe the standard is to capitalize the component files since you capitalize custom components in React. Should we update our filenames real quick?